.github/workflows/cd.yml

```diff
@@ -4,29 +4,81 @@ on:
     branches: [develop]
 
 jobs:
-  release-next:
-    name: release-next
-    runs-on: ubuntu-latest
+  test:
+    runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2.3.4
-        with:
-          fetch-depth: "0"
-      - run: git fetch origin +refs/tags/*:refs/tags/*
       - name: Setup Node.js environment
         uses: actions/setup-node@v2.1.4
         with:
           node-version: 14.x
-          registry-url: https://registry.npmjs.org/
       - name: Install dependencies
-        run: bash ./.build/setup_${{ runner.os }}.sh
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y gnome-calculator
+          docker pull selenium/standalone-chrome-debug
+          docker pull selenium/standalone-firefox-debug
+          docker run -d -p 4444:4444 selenium/standalone-firefox-debug
+          docker run -d -p 4445:4444 selenium/standalone-chrome-debug
       - run: npm ci
       - run: npm run build
-      - name: Run unit/integration tests
+      - run: npm run coverage:clean
+      - name: run unit tests
         uses: GabrielBB/xvfb-action@v1
         with:
-          run: bash ./.build/run_${{ runner.os }}.sh
-      - name: Run e2e tests
+          run: npm test -- --runInBand --ci --bail
+        env:
+          FIREFOX_WD_URL: http://localhost:4444/wd/hub
+          CHROME_WD_URL: http://localhost:4445/wd/hub
+      - name: run integration tests
+        uses: GabrielBB/xvfb-action@v1
+        with:
+          run: npm run test:it
+      - name: Merge coverage report
+        run: |
+          npm run coverage:merge
+          npm run coverage:merge-report
```

Comment:
```suggestion
```
Sorry, just saw that. We do not need a coverage merge on cd as we don't send it to sonar cloud anyways.